examples/multi-language/README.md

```diff
@@ -126,9 +126,25 @@ gsutil cat gs://$GCP_BUCKET/multi-language-beam/output*
 
 #### Instructions for running the Java pipeline at HEAD (Beam 2.41.0 and 2.42.0).
 
+* Activate a new virtual environment following
+[these instructions](https://beam.apache.org/get-started/quickstart-py/#create-and-activate-a-virtual-environment).
+
+* 2. Install Apache Beam package with gcp support and the `sklearn` package.
+
+```
+pip install apache-beam[gcp]
+pip install sklearn
+```
+
+* Startup the expansion service
+
+```
+python -m apache_beam.runners.portability.expansion_service_main -p <PORT> --fully_qualified_name_glob "*"
```

Comment:
Do we need --pickle_library=cloudpickle ?

Reply:
I don't think we can add it till https://github.com/apache/beam/pull/23739 is released :(.

After that we can update all guides/examples.